packages/os/cfsignal.service

```
@@ -0,0 +1,13 @@
+[Unit]
+Description=Send signal to CloudFormation Stack
+After=network-online.target settings-applier.service
+Requires=settings-applier.service
+Wants=network-online.target
+
+[Service]
+Type=simple
```

```suggestion
Type=oneshot
```

I think `oneshot` makes more sense for this service, even though we don't yet have any services that depend on it.

---

from systemd.service man page:
* oneshot
```
Behavior of oneshot is similar to simple; however, the service manager will consider the unit started
after the main process exits
```
* simple
```
If set to simple.... , the service manager will consider the unit started
immediately after the main service process has been forked off
```
i cannot wait that process exit, because i am calling systemctl with `--wait` option inside the program itself, i had not tested it but i think that boot process could remain stuck.

---

> i cannot wait that process exit, because i am calling systemctl with --wait option inside the program itself, i had not tested it but i think that boot process could remain stuck.

Got it, thanks for the explanation!